unittests/CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ set(unittests
   unittest_mtrie
   unittest_ip_resolver
   unittest_udp_address
+  unittest_radix_tree
+  #  unittest_trie
+  #  unittest_trie_gb
```

Comment:
These are no unit tests at all, but performance measurements, they should be renamed. 

It is not desirable to have code that is never built. I think it would be better to conditionally build them if the toolset is c++11 compliant. 

But probably this is not the right place, and they should not be run along with the unittests.